extensions/oplogPopulator/modules/ConnectorsManager.js

```diff
@@ -312,6 +352,7 @@ class ConnectorsManager {
      */
     scheduleConnectorUpdates() {
         schedule.scheduleJob(this._cronRule, async () => {
+            await this._validateConnectorsState();
```

Comment:
does it make sense to check the status of a connector we update? (both just before or after the update)
→ maybe we should only check for restart in the `else` case of `updateConnectors` (line 283)?

Reply:
Checking each connector state before updating it makes more sense yes, but i think it should be done systematically, no just when we don't update the connector